install/app-neovim.sh

```diff
@@ -1,6 +1,11 @@
-sudo add-apt-repository -y ppa:neovim-ppa/stable
-sudo apt update
-sudo apt install -y neovim
+git clone https://github.com/neovim/neovim
```

Comment:
May be `cd` into the Downloads folder first?

Reply:
Yeah, we can de definitely do that. I updated my code, we also remove the repository after the installation and get back to `~/`.